apistar/backends/sqlalchemy.py

```diff
@@ -16,7 +16,7 @@ def build(cls, settings: Settings):
         metadata = config.get('METADATA')
 
         kwargs = {}
-        if url.startswith('postgresql:'):  # pragma: no cover
+        if url.startswith('postgresql'):  # pragma: no cover
```

Comment:
Idealt we have a set of all the values that we *do* want to include pool size on.